lua/dap.lua

```diff
@@ -1357,6 +1464,30 @@ function M.set_exception_breakpoints(filters, exceptionOptions)
 end
 
 
+function M.clear_data_breakpoints()
+  data_bp_info = {}
+  if not session then return end
+  session:_set_data_breakpoints()
+end
+
+
+function M.toggle_data_breakpoint(resolve_expression_fn, condition, hit_condition)
```

Comment:
The DAP protocol allows data breakpoints to be an arbitrary expression. But I don't use an adapter that support this.